packages/hardhat-plugin/src/index.ts

```diff
@@ -206,6 +212,64 @@ task("plan")
     }
   );
 
+task("ignition-info")
+  .addPositionalParam("moduleNameOrPath")
+  .setDescription("Lists the status of all deployments")
+  .setAction(
+    async ({ moduleNameOrPath }: { moduleNameOrPath: string }, hre) => {
+      const userModule: Module<ModuleDict> | undefined = loadModule(
+        hre.config.paths.ignition,
+        moduleNameOrPath
+      );
+
+      if (userModule === undefined) {
+        console.warn("No Ignition modules found");
+        process.exit(0);
+      }
+
+      const journalPath = resolveJournalPath(
+        userModule?.name,
+        hre.config.paths.ignition
+      );
+
+      const deployments = await hre.ignition.info(userModule.name, journalPath);
```

Comment:
Can we narrow this api between core and the plugin.

If the code here is moved down into core so that only the final calculated network to deployed contract names/addresses mapping is passed back.